build/AzureInfo.props

```diff
@@ -8,7 +8,14 @@
     <ArtifactCloudDropAccountName Condition="'$(ArtifactCloudDropAccountName)' == ''">dotnetcli</ArtifactCloudDropAccountName>
     <DotnetBlobRootUrl>https://$(ArtifactCloudDropAccountName).blob.core.windows.net/$(ArtifactContainerName)</DotnetBlobRootUrl>
     <ChecksumContainerName>$(CHECKSUM_STORAGE_CONTAINER)</ChecksumContainerName>
+    <ChecksumContainerName Condition="'$(ChecksumContainerName)' == ''">dotnet</ChecksumContainerName>
     <ChecksumCloudDropAccessToken>$(CHECKSUM_STORAGE_KEY)</ChecksumCloudDropAccessToken>
     <ChecksumCloudDropAccountName>$(CHECKSUM_STORAGE_ACCOUNT)</ChecksumCloudDropAccountName>
+    <ChecksumCloudDropAccountName Condition="'$(ChecksumCloudDropAccountName)' == ''">dotnetclichecksums</ChecksumCloudDropAccountName>
+    <TransportFeedContainerName>$(TRANSPORTFEED_STORAGE_CONTAINER)</TransportFeedContainerName>
```

Comment:
Is this pattern determining the transport feed values via environment variables?  Why not just specify them explicitly rather than relying on environment variables?  My preference is to avoid environment stuff because it makes repro's difficult (and debugging via logs)

Reply:
We can't have the values in the open in our build scripts right? Otherwise anyone will be able to copy and push to our blob storage.

Reply:
They would be supplied explicitly via `/p:foo=bar`

Reply:
They indeed are supplied explicitly via properties thru the VSO build definition. Please see the initial comments...

Reply:
But why pass them as environment variables, why not just use the property directly?

Reply:
We are not passing them as environment variables. From: https://devdiv.visualstudio.com/DevDiv/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=6786

Here are the args to 'build.cmd' in the "Publish Build Artifacts" step:
/p:Configuration=$(BuildConfiguration) /t:Publish  /p:ARTIFACT_STORAGE_KEY=$(ARTIFACT_STORAGE_KEY) /p:CHECKSUM_STORAGE_KEY=$(CHECKSUM_STORAGE_KEY) /p:GITHUB_PASSWORD=$(GITHUB_PASSWORD)

Reply:
Ah, I guess that was the piece I was missing.  I'm still confused by the level of indirection though.  ie, why not `/p:TransportFeedContainerName=$(TRANSPORTFEED_STORAGE_CONTAINER)`?

Reply:
"TransportFeedContainerName" is not a secret and we have a default value of "dotnet-core". If someone [like me for example] wants to test a different location, like: "jbeisner" - then the command line would have: /p:TransportFeedContainerName=jbeisner  I have done exactly this and it seems work well: https://dotnetfeed.blob.core.windows.net/jbeisner/packages/microsoft.dotnet.cli.utils/2.0.3-servicing-007047/microsoft.dotnet.cli.utils.2.0.3-servicing-007047.nupkg

Reply:
I'm still not understanding the indirection, to me it just complicates things (ever so slightly).  In my mind `/p:TransportFeedContainerName=jbeisner` and `/p:TransportFeedContainerName=$(TRANSPORT_STORAGE_CONTAINER)` would be exactly how I would expect this to work.  I'm not certain what the extra layer provides since it does nothing to really making anything more or less secret.  It only means that the scripts define it one way, and a human defines it a different way.